libkbfs/journal_server.go

```diff
@@ -280,6 +280,67 @@ func (j *JournalServer) hasTLFJournal(tlfID tlf.ID) bool {
 	return ok
 }
 
+func (j *JournalServer) makeFBOForJournal(
+	ctx context.Context, tj *tlfJournal, tlfID tlf.ID) error {
+	bid, err := tj.getBranchID()
+	if err != nil {
+		return err
+	}
+
+	head, err := tj.getMDHead(ctx, bid)
+	if err != nil {
+		return err
+	}
+
+	if head == (ImmutableBareRootMetadata{}) {
+		return nil
+	}
+
+	headBareHandle, err := head.MakeBareTlfHandleWithExtra()
+	if err != nil {
+		return err
+	}
+
+	handle, err := MakeTlfHandle(
+		ctx, headBareHandle, tlfID.Type(), j.config.KBPKI(),
+		j.config.KBPKI(), constIDGetter{tlfID})
+	if err != nil {
+		return err
+	}
+
+	_, _, err = j.config.KBFSOps().GetRootNode(ctx, handle, MasterBranch)
```

Comment:
This will also trigger the prefetcher for all existing journals. Should be fine, just something to be aware of.